wayland-protocols/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Breaking Change
+
+- Complete reorganization of the crate around the `wp`/`xdg`/`wp` categories
```

Comment:
wp/xdg/ext?

Reply:
More like wg/xdg/ext/kde? Or we don't want to bundle kde protocols? or just put them in ext category?

Reply:
@i509VCB Indeed, that's a bad copy-paste.

@HeavyRain266 The point of this PR is to split wayland-protocols into multiple crates, so that the `wayland-protocols` crate itself only contains the extensions from the official wayland-protocols upstream repo. Those 3 categories are those from that upstream.

If We are to provide bindings for KDE protocols, they should be maybe in the `wayland-protocols-misc` crate introduced by this PR, or as a `wayland-protocols-kde` crate. However, I'd like to get some clarifications about how much of a public API these protocols are considered by KDE. I'm not willing to take the responsibility of maintaining public bindings to a private API.

Reply:
I know that from matrix, asked about KDE because there is PR which implements public "idle" protocol used by popular compositors.

Reply:
Well, if that protocol is already widely used, it would go in the `wayland-protocols-misc` crate then.

Reply:
Ok so, having found the official KDE protocols repo, I'm making a `wayland-protocols-plasma` crate to track it. With a few caveats because some KDE protocols are a little too wonky for `wayland-scanner` to parse.